app/client/packages/design-system/widgets/src/components/ButtonGroup/src/styles.module.css

```
@@ -0,0 +1,123 @@
+@import "../../../shared/colors/colors.module.css";
+
+.buttonGroup {
+  display: flex;
+
+  &[data-orientation="vertical"] {
+    flex-direction: column;
+  }
+
+  /* increasing z index to make sure the focused button is on top of the others */
+  & [data-button]:not([data-disabled]):focus {
```

```suggestion
  & [data-button]:not([data-disabled]):focus-visible {
```

So that it applied only on keyboard focus and not after a mouse click.

---

+1

---

fixed